README.md

```diff
@@ -169,6 +152,20 @@ Allow this process to continue running in its own terminal window or `tmux` sess
 
 TODO: Add tutorials.
 
+## Quick Alternative Method to Start the Container
+
+This alternative uses docker volumes to persist the Unchained Index and binary caches.
+This method is great if you plan to always use docker to run chifra or to quickly
+try out chifra.
```

Comment:
Add "...and you don't need the Unchained Index files externally."

Or some such wording. Make it more clear that this version the index files are inside docker.